### PR TITLE
updated meta tags, added og tags

### LINKED ISSUE #33: Improve SEO for Website
Add meta tags, og attributes etc.
Enable indexing for search engines, use robots.txt

### PR DESCRIPTION
**Pull request by bhansa**
Fixes #33

### CODE REVIEW

File: _includes/head.html [hunk 1]
```diff
@@ -1,14 +1,32 @@
...
+	<!-- Open Graph / Facebook -->
+	<meta property="og:type" content="website">
+	<meta property="og:url" content="https://pyconf.hydpy.org/2019/">
+	<meta property="og:title" content="{{ title }}">
+	<meta property="og:description" content="{{ site.description }}">
+	<meta property="og:image" content="assets/images/pyconflogo_ai_compressed.jpg">
+
+	<!-- Twitter -->
+	<meta property="twitter:card" content="summary_large_image">
+	<meta property="twitter:url" content="https://pyconf.hydpy.org/2019/">
```
Comment (ananyo2012) [inline _includes/head.html:27]: Should be the @pyconfhyd twitter url

Comment (bhansa) [reply _includes/head.html:27]: This seems to be correct Reference: https://davidwalsh.name/twitter-cards
I should add the twitter account details though in separate meta tag.

Comment (bhansa) [reply _includes/head.html:27]: Added `twitter:site` and `twitter:creator` details.

Comment (ananyo2012) [reply _includes/head.html:27]: Got it thanks. So this data will be used by twitter for preview

**Comment by ananyo2012** [comment]
LGTM. Merging this in.

**Pull request merged**